tests/optim/models/test_models.py

```diff
@@ -70,7 +66,6 @@ def test_load_inceptionv1_linear(self) -> None:
         _check_layer_not_in_model(self, model, torch.nn.ReLU)
         _check_layer_not_in_model(self, model, torch.nn.MaxPool2d)
         _check_layer_in_model(self, model, SkipLayer)
-        _check_layer_in_model(self, model, AvgPool2dConstrained)

```

Comment:
You should make sure to test for the average pooling layer like this:

```
_check_layer_in_model(self, model, torch.nn.AvgPool2d)
```

Reply:
Fixed!